pkg/codegen/go/gen_program_test.go

```diff
@@ -28,6 +28,11 @@ func TestGenProgram(t *testing.T) {
 			continue
 		}
 
+		if filepath.Base(f.Name()) == "azure-native.pp" {
```

Comment:
This Go program generation already fails in master. This PR doesn't fix it but it doesn't make it worse either, so I suggest filing a separate issue for later.

Reply:
SGTM